tools/intrinsic_db/advsimd_classification.csv

```diff
@@ -4389,4 +4515,55 @@ vbfmlaltq_f32	Vector arithmetic|Multiply|Multiply-accumulate
 vbfmlalbq_lane_f32	Scalar arithmetic|Vector multiply-accumulate by scalar
 vbfmlalbq_laneq_f32	Scalar arithmetic|Vector multiply-accumulate by scalar
 vbfmlaltq_lane_f32	Scalar arithmetic|Vector multiply-accumulate by scalar
-vbfmlaltq_laneq_f32	Scalar arithmetic|Vector multiply-accumulate by scalar
\ No newline at end of file
+vbfmlaltq_laneq_f32	Scalar arithmetic|Vector multiply-accumulate by scalar
+vcvt1_bf16_mf8_fpm	Conversion|Convert to BFloat16 (vector, lower)
+vcvt1_low_bf16_mf8_fpm	Conversion|Convert to BFloat16 (vector, lower)
+vcvt2_bf16_mf8_fpm	Conversion|Convert to BFloat16 (vector, lower)
+vcvt2_low_bf16_mf8_fpm	Conversion|Convert to BFloat16 (vector, lower)
+vcvt1_high_bf16_mf8_fpm	Conversion|Convert to BFloat16 (vector, upper)
+vcvt2_high_bf16_mf8_fpm	Conversion|Convert to BFloat16 (vector, upper)
+vcvt1_f16_mf8_fpm	Conversion|Convert to half-precision (vector, lower)
+vcvt1_low_f16_mf8_fpm	Conversion|Convert to half-precision (vector, lower)
+vcvt2_f16_mf8_fpm	Conversion|Convert to half-precision (vector, lower)
+vcvt2_low_f16_mf8_fpm	Conversion|Convert to half-precision (vector, lower)
+vcvt1_high_f16_mf8_fpm	Conversion|Convert to half-precision (vector, upper)
+vcvt2_high_f16_mf8_fpm	Conversion|Convert to half-precision (vector, upper)
+vcvt_mf8_f32_fpm	Conversion|Convert single-precision to floating point (vector, lower)
```

Comment:
is the description correct here? shouldn't it state that this is conversion to FP8 type?